airbyte-integrations/bases/base-normalization/integration_tests/normalization_test_output/postgres/test_primary_key_streams/final/airbyte_ctes/test_normalization/nested_stream_with_c___long_names_partition_ab1.sql

```diff
@@ -6,6 +6,7 @@ select
     _airbyte_nested_stre__nto_long_names_hashid,
     jsonb_extract_path("partition", 'double_array_data') as double_array_data,
     jsonb_extract_path("partition", 'DATA') as "DATA",
+    jsonb_extract_path("partition", 'column`_''with"_quotes') as "column`_'with""_quotes",
```

Comment:
Postgres is able to parse the json blob just fine. It just has an edge case of doubling the `'` characters to "escape" it...